lib/fxa.js

```diff
@@ -0,0 +1,35 @@
+"use strict";
+
+const got = require("got");
+const { URL } = require("url");
+
+const AppConstants = require("../app-constants");
+const mozlog = require("../log");
+
+
+const log = mozlog("fxa");
```

Comment:
I [searched for current "mozlog" usage](https://github.com/mozilla/blurts-server/search?q=mozlog&type=Code), but this may be the first instance in lib/*. But for controllers/ses.js, we used `const log = mozlog("controllers.ses");` (but possibly only to differentiate from `const log = mozlog("routes.ses")`).

Not sure if we want to use `mozlog("lib.fxa")` for consistency, or if that is the tiniest of papercuts.

Reply:
Good catch, but I'll fix it in a future PR.

I'm about to move a bunch of modules into `lib/` and I think the module names will all be unique enough to know they are `lib` modules. But if it looks like we'll have module names that are common across `controllers/` and `lib/` I'll update the `mozlog` calls.